docs/js-core-2/week-1/lesson.md

```diff
@@ -1258,6 +1259,165 @@ console.log(
+
+Discuss as a class:
+
+- Why is it important for your team to understand their main goal?
+- Why is it important for everybody on your team to understand the role each other play?
+- What happens when you are not clear about these two points?
+
+:::
+
+:::note
+Exercise (5 minutes)
+
+**Exercise objective:** To get familiar with different processes in the software development cycle
+Match the process to the stage in the software development cycle (SDC) as a class. Don’t go too much into detail, you can do some research later if you would like to learn more. These processes vary from company to company, and determine what steps need to happen before code is shipped to production. They aim to reduce the risk of mistakes going to production. For example during the deployment process, code is reviewed by at least one person, deployed to a testing environment (server), successfully tested by a QA, and the release is scheduled to a time approved by the end customer.
+
+Process:
+
+a. Gathering Requirements
+
+b. Code Review
+
+c. Bug Fixes
+
+d. Deployment Processes
+
+Software Development Cycle:
+
+Planning
+
+Analysis
+
+Design
+
+Implementation
+
+Testing&Integration
+
+Maintenance
+
+What other processes can you think of?
+
+**Answer Key**
+
+Gathering Requirements - Analysis and planning
+
+Code Review - Implementation
+
+Bug Fixes - Testing and maintenance
+
+Deployment Processes - Integration
+
+:::
+
+:::note
+Exercise (10  minutes)
+
+**Exercise objective:** To identify the most common time management challenges when working as a team
+Work in groups for about 5 minutes, then discuss as a class for 5 minutes:
+
+- How will you ensure that your group will remain on schedule?
+- What is your strategy if the project falls behind schedule? 
+
+:::
+
+:::note
+Exercise (10  minutes)
+
+**Exercise objective:** To recognise the value of co-working tools
+
+Work in groups for about 5 minutes, then discuss as a class for 5 minutes:
+
+- What is your strategy for communication within your team?
+- How would you work with your team to agree on task/workload allocation? What tools will you use?
+
+**Answer Key**
+For the second question the correct answer at CYF is “GitHub Projects”.
+Some examples for tools: Jira, GitHub (repo and project boards), IDE (integrated development environment), Slack, Teams, Google calendar, Calendly, Microsoft Office, Outlook, shared documents, Miro, Figma, Jamboard
+
+:::
+
+:::note
+Exercise (10 minutes)
+
+**Exercise objective:** To brainstorm about the causes of conflicts and methods to manage them
+Remember, there will be a more detailed PD session on conflict resolution in the upcoming weeks. Work in groups for about 5 minutes, then discuss as a class for 5 minutes:
```

Comment:
```suggestion
**Exercise objective:** To brainstorm about the causes of conflicts and methods to manage them

Remember, there will be a more detailed PD session on conflict resolution in the upcoming weeks. Work in groups for about 5 minutes, then discuss as a class for 5 minutes:
```

If there's not an empty line between two lines in markdown, the two lines get folded into one.